Dockerfile

```diff
@@ -37,9 +37,19 @@ ENV GIT_VERSION=1:2.39.2-1.1
 
 # Install from backports since renovate requires at least git 2.33.0
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends git=${GIT_VERSION} && \
+    apt-get install -y --no-install-recommends curl git=${GIT_VERSION} && \
```

Comment:
This will result having curl in the final image which is not required. To avoid this image needs to be changed to a multi-stage build like it is for example done here: https://github.com/swissgrc/docker-azure-pipelines-dockercli/blob/develop/Dockerfile